optimade/validator/validator.py

```diff
@@ -1233,6 +1236,23 @@ def _test_bad_version_returns_553(self) -> None:
             "v123123", expected_status_code=expected_status_code, optional=True
         )
 
+    @test_case
+    def _test_meta_schema_reporting(
+        self,
+        response: requests.models.Response,
+        request_str: str,
+    ):
+        """Tests that the endpoint responds with a `meta->schema`."""
+        if not response.json().get("meta", {}).get("schema"):
```

Comment:
Would it be worthwhile to test the schema field further? For example, by testing whether it is a valid url, and that there indeed is a schema at that url?